role handbooks: add kubernetes-release-team-shadows list (#1090)
kubernetes/sig-release: release-team/role-handbooks/emeritus-adviser/README.md

4. Close the Shadow Application around 7-10 days after the initial announcement.
5. Chat with each of the incoming Role Leads about selecting their shadows, and make sure that they stay on schedule for it.
6. Once all role leads select their shadows the EA will send out a notification to all applicants not selected for this release cycle 

> 7. Ensure the outgoing EA and SIG Release co-chairs have a swizzled you in as an owner of the [kubernetes-release-team-shadows](https://groups.google.com/forum/#!forum/kubernetes-release-team-shadows) Google Group.

Reviewer: ```suggestion
7. Ensure the outgoing EA or one of the SIG Release co-chairs has added you as an owner of the [kubernetes-release-team-shadows](https://groups.google.com/forum/#!forum/kubernetes-release-team-shadows) Google Group.
```

Author: fixed

Weeks: 1 to 12

> Remove prior release shadows from the [kubernetes-release-team-shadows](https://groups.google.com/forum/#!forum/kubernetes-release-team-shadows) Google Group and add the current release shadows.  This list can facilitate easier communication between what at times is 2-3 dozen shadows.

Reviewer: ```suggestion
Remove prior release shadows from the [kubernetes-release-team-shadows](https://groups.google.com/forum/#!forum/kubernetes-release-team-shadows) Google Group and add the current release shadows. This list can facilitate easier communication between what at times is 2-3 dozen shadows.
```

Reviewer: Add line breaks for easier future reviews.

Author: I'd meant to not add line breaks since the file didn't have them and that seemed to be the prevailing style.  If there's a strong preference on things like this can they be put in a linter?

Reviewer: Ahhh. Missed that! Definitely prefer the prevailing style.
Eventually yes, we should lint all the things!

Author: Sohweet I think two spaces is prevailing too across the history of computer/typed text ;)

> Poll the new shadows (eg: [Doodle](Doodle.com)) for their availability to have a "shadow orientation" meeting sometime in week 2 or week 3, after all shadows are selected.  This meeting will likely have to be split for time zone reasons and held twice.  In this meeting, go over the information in the main Shadow docs, give information about the release team and SIG Release, and then open things for questions or discussion. Make sure that Shadows know they can come to you if they have problems during their apprenticeship.

Reviewer: ```suggestion
Poll the new shadows (e.g., via [Doodle](https://doodle.com)) for their availability to have a "shadow orientation" meeting sometime in week 2 or week 3, after all shadows are selected. This meeting will likely have to be split for time zone reasons and held twice.  In this meeting, go over the information in the main Shadow docs, give information about the Release Team and SIG Release, and then open things for questions or discussion. Make sure that Shadows know they can come to you if they have problems during their apprenticeship.
```

Reviewer: Add line breaks for easier future reviews.
